app/components/password/reset-password.js

```diff
@@ -11,5 +12,35 @@ export default Component.extend({
    * @property passwordConfirmation
    * @default String
    */
-  passwordConfirmation: ''
+  passwordConfirmation: '',
+  /**
+   * @property error
+   */
+  error: null,
+
+  /**
+   * @property resetPasswordTask
```

Comment:
For clearing errors, I would just clear it in the `yield`. As in, when you start the task again, clear the errors from the previous task.